src/Templates/OrchardCore.ProjectTemplates/content/OrchardCore.Templates.Cms.Web/OrchardCore.Templates.Cms.Web.csproj

```diff
@@ -11,6 +11,11 @@
     <Folder Include="Localization\" />
   </ItemGroup>
 
+  <!-- Watcher include and excludes -->
+  <ItemGroup>
+      <Watch Include="**\*.cs" Exclude="Recipes\**;Assets\**;node_modules\**\*;**\*.js.map;obj\**\*;bin\**\*" />
```

Comment:
Why Include and Exclude attributes?
`Include="**\*.cs"` does it mean that they are not watched otherwise, but then the Exclude would be useless.
What about cshtml, liquid, ?

Reply:
That's because it requires an include and exclude to work. Else it complains about it missing an include so I added the a value already present in the defaults. Here is the documentation about this : https://docs.microsoft.com/en-us/aspnet/core/tutorials/dotnet-watch?view=aspnetcore-6.0#customize-files-list-to-watch